teams/wg-secure-code.toml

```diff
@@ -7,12 +7,18 @@ members = [
     "Shnatsel",
     "tarcieri",
     "alex",
+    "pinkforest",
+]
+alumni = [
     "danielhenrymantilla",
     "stusmall",
     "xacrimon",
-    "pinkforest",
 ]
 
+[[github]]
+team-name = "wg-secure-code"
+orgs = ["rust-lang"]
```

Comment:
I think `orgs` should be `rust-secure-code` and `rustsec`

I'm not in the rust-lang org for example

Reply:
This just source of truth for rust-lang so our repo's here

Reply:
Oh, this repo is the source of truth. Nevermind then.